.github/workflows/automaton.yml

```
@@ -0,0 +1,155 @@
+# Morty, a beepsky supplement for checking runtime violations in maps
+# Based on Turdis by Yogstation
+
+name: Morty
+
+on:
+  pull_request:
+    branches: master
+  push:
+
+jobs:
+
+  runtime:
+    name: Runtime Checker
+    runs-on: ubuntu-18.04
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    strategy:
+      fail-fast: false
+      matrix:
+        map: ["COGMAP", "COGMAP2", "CLARION", "DESTINY", "DONUT2", "DONUT3", "HORIZON", "MUSHROOM", "TRUNKMAP", "OSHAN", "KONDARU", "FLEET", "ICARUS", "DENSITY", "ATLAS", "MANTA", "WRESTLEMAP", "SPACE_PREFAB", "UNDERWATER_PREFAB", "FULL"]
```

```suggestion
        map: ["COGMAP", "COGMAP2", "CLARION", "DESTINY", "DONUT3", "HORIZON", "OSHAN", "KONDARU", "ATLAS", "MANTA", "SPACE_PREFAB", "UNDERWATER_PREFAB", "FULL"]
```
removing assjam-only maps, fuck them